### ISSUE TITLE
Sequence Function Maps with the same colorbar range

### ISSUE THREAD

**Issue by giameier** [FowlerLab/Enrich2#33]
Hi
Is there a way to use the same colorbar range in the Sequence function map over all conditions for an experiment. I see that there is an option in  sfmap_axes to pass vmax and vmin as arguments. How can I define these parameters for an experiment?
Thank you in advance.

**Comment by afrubin** [comment]
I don't think this is currently something that can be set, but it's a good feature request.
